.github/dependabot.yml

```diff
@@ -6,6 +6,16 @@ updates:
     interval: "daily"
     time: "09:00"
     timezone: "America/Los_Angeles"
+  groups:
```

Comment:
may we add?
-  github.com/Azure/azure-sdk-for-go
- github.com/Azure/go-autorest

Reply:
Added azure-sdk-for-go. I don't think we need to add go-autorest since that is an indirect dep.